readthedocs/builds/querysets.py

```diff
@@ -119,6 +119,9 @@ def public(
     def api(self, user=None):
         return self.public(user, only_active=False)
 
+    def api_v2(self, *args, **kwargs):
+        return self.api(*args, **kwargs)
```

Comment:
It would be good to have a comment on these methods that explains the differences between `.api` and `.api_v2`. Even reading the current context from the issues, it's still hard to have everything in mind.